examples/README.md

```diff
@@ -455,4 +480,29 @@ Device name is: AMD Radeon PRO W6800
 Running with single precision real data type:
 DFT example run_time dispatch
 DFT example ran OK
+```
+
+```none
+$ LD_LIBRARY_PATH=lib ./bin/example_dft_real_fwd_usm 
+########################################################################
+# DFTI complex in-place forward transform with USM API example:
+#
+# Using APIs:
+#   USM forward complex in-place
+#   Run-time dispatch
+#
+# Using single precision (float) data type
+#
+# Device will be selected during runtime.
+# The environment variable SYCL_DEVICE_FILTER can be used to specify
+# SYCL device
+#
+########################################################################
+
+Running DFT complex forward example on GPU device
+Device name is: Intel(R) UHD Graphics 750
+Running with single precision real data type:
+DFT example run_time dispatch
+Caught std::exception:
+	oneMKL: dft/backends/portfft/commit: function is not implemented portFFT only supports complex to complex transforms
```

Comment:
Is it really useful to add an example where portFFT does not support the configuration? I don't really get the point of adding example output to begin with.

Reply:
I was kind of just following what the other backends had done. I supposed the example output might be helpful to check that you are seeing the correct thing.

I could just not have the example. I'm sure uses can figure it out from existing examples.

Reply:
actually, this is the runtime dispatch example so I don't want to modify that example. 
All the other backends have their output here, so maybe it's silly but it's honest

Reply:
comptime example removed here: https://github.com/FMarno/oneMKL/pull/1/commits/fb0ad2f9f2d9c9b8aa630c3eaa2b58dee04f5d60